x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/index.tsx

```diff
@@ -444,6 +439,15 @@ export const AddExceptionFlyout = memo(function AddExceptionFlyout({
     return hasOsSelection && selectedOs === undefined;
   }, [hasOsSelection, selectedOs]);
 
+  const allowLargeValueLists = useMemo(
+    () =>
+      maybeRule?.type === 'query' ||
```

Comment:
nit: might be cleaner to have these in an array somewhere and then reuse that with .includes in this file and the edit exception flyout